### ISSUE TITLE
Feature Request: Visible name of Task you're on

### ISSUE THREAD

**Issue by zsection** [minbrowser/min#300]
Woud find it incredibly useful to be able to tell what task I'm actually on.

**Comment by doreminiature** [comment]
One possible solution will be to add task title in Min window title bar. This is for windows and Linux.
For macs or for all systems at once put the task title to the right of overlay menu icon. This is obvious solution.

**Comment by clayton-grey** [comment]
New user here, but this was something I was hoping for pretty quickly. Some way to see the current task without loading the TO would be _very_ useful! Either temporarily or permanently. Just a way to read it easily once the UI disappears. 

If I tap the hot key for switching rather than enter the TO with a mouse and manually select, the brief moment the TO is visible isn't really enough time to make sense of what is happening with multiple tasks with multiple tabs. I don't always want to consider all tasks, so much as switch between the ones I have open. Relates to #136 and #469.

**UI suggestion:** 

- A small overlay popover underneath the TO button that fades after a couple of seconds. 
- A permanent task name UI seems problematic as it could require considerable space in the tab bar, which seems very unwelcome. 
- Mouse hover on the TO button could also restore the overlay if a permanent UI element for task name is undesirable. 
- You could eliminate showing the TO when using hotkeys with this technique. So long as we can see what task we're entering without contextualizing the content of the tab, that feels like it would be a nice improvement!

**Comment by clayton-grey** [comment]
Yeah, the second is closer. That said, it seems like option 2 is a better solution that avoids needing to mess with the browser view, as you can merely hide the tabs for a beat or two. A hotkey to show show the current task name view would be a nice bonus. ☺️ 

As is I still end up opening the TO pretty often just to "orient" myself. That's fine when I want to organize things, but it's a drag when I'm just trying to hop between tasks quickly. 

As far a keys, perhaps +SHIFT to cycle in the TO, sans SHIFT to cycle using only the tab name view?

**Comment by PalmerAL** [comment]
Regarding the keys, I think the best option would be to do the same thing that cmd+tab does on macOS (and I think its the same on other operating systems): if you press the keys quickly, the next task will appear immediately, but if you hold them down longer the overlay will show up. I'm not sure if we can actually do that though with the key events we get; if we can't, having two separate shortcuts would probably work as well.

**Comment by clayton-grey** [comment]
Either interaction could work! ☺️

**Comment by a13ph** [comment]
Was implemented recently in a way. Task name is displayed when title bar is enabled
